src/modules/CVs/api/index.ts

```diff
@@ -0,0 +1,53 @@
+import { gql } from "@apollo/client";
+
+export const CVS = gql`
```

Comment:
May be i can rename folder mutation to hooks and it will be good?

Do we really need word "QUERY" in each variable name?

and why should we add "GET" to all get request. 
We can negotiate that all queries without type is get queries.
i think  it can improve readability 
![image](https://github.com/savelievamareena/cv-final/assets/70685916/f6f67ac7-e3cf-4c4c-a356-6a67116eabd3)

Reply:
ok, sound reasonable. Will be reorganized